mm2src/coins/utxo/rpc_clients.rs

```diff
@@ -1883,6 +1916,84 @@ impl ElectrumClient {
     pub fn blockchain_transaction_get_merkle(&self, txid: H256Json, height: u64) -> RpcRes<TxMerkleBranch> {
         rpc_func!(self, "blockchain.transaction.get_merkle", txid, height)
     }
+
+    async fn get_tx_height(&self, tx: &UtxoTx) -> Result<u64, MmError<GetTxHeightError>> {
+        for output in tx.outputs.clone() {
+            let script_pubkey_str = hex::encode(electrum_script_hash(&output.script_pubkey));
```

Comment:
Instead of requesting the history of some address (that can be very large), it should be possible to get verbose tx and find the corresponding block height in DB by `block_hash`. This way, you can also pass transaction hash as an argument instead of entire tx.

Reply:
Done as I suggested above.

Reply:
I assumed that this fn will be removed at all 🙂 I think it should be possible if this change is implemented https://github.com/KomodoPlatform/atomicDEX-API/pull/1339#discussion_r924326029